packages/api/src/external/carequality/command/cq-directory/get-cq-directory-entries.ts

```diff
@@ -24,15 +24,16 @@ export async function getHieDirectoryEntriesByFilter({
       ? ` AND (search_criteria @@ websearch_to_tsquery('english', :filter) OR id = :filter)`
       : "");
 
-  const queryFinal = queryFTS + paginationSqlExpressions(pagination);
+  const { query: paginationQueryExpression, replacements: paginationReplacements } =
+    paginationSqlExpressions(pagination);
+  const count = paginationReplacements.count;
+  const queryFinal = queryFTS + paginationQueryExpression;
 
-  const { toItem, fromItem, count } = pagination ?? {};
   const cqDirectoryEntries = await sequelize.query(queryFinal, {
     model: HIEDirectoryEntryViewModel,
     mapToModel: true,
     replacements: {
-      ...(toItem ? { toItem } : {}),
-      ...(fromItem ? { fromItem } : {}),
+      ...paginationReplacements,
       ...(count ? { count } : { count: defaultPageSize }),
```

Comment:
This already comes from `paginationSqlExpressions`. We could add an optional parameter to that function to determine the default count, so we can customize it per endpoint (I'd actually not worry too much about having diff defaults for count until we need that, though, and assume 50 throughout).

Reply:
ahh ok yeah thats fair, yeah ill just move the default inside the helper

Reply:
addressed